config/config.yml

```
@@ -23,6 +23,7 @@ gdas:
   file_pattern: "gfs.t{hour:02d}z.pgrb2.0p25.anl"
```

Seems like this template may need to be updated to include the placeholder for resolution?

---

Yea wasn't sure exactly how to do that without using jinja or something else

---

Seems `file_pattern.format(hour=hour, resolution=res_str)` is how it's used, after the replacing `.` with `p`. So couldn't this work?
```suggestion
  file_pattern: "gfs.t{hour:02d}z.pgrb2.{resolution}.anl"
```